cpp/include/raft/neighbors/ball_cover-ext.cuh

```diff
@@ -121,4 +167,9 @@ void knn_query(raft::resources const& handle,
 
 instantiate_raft_neighbors_ball_cover(int64_t, float, uint32_t, uint32_t);
 
+instantiate_raft_neighbors_ball_cover(int32_t, float, int32_t, int32_t);
```

Comment:
Do you have an idea of how much all these new instantiations are increasing the binary size? We've been watching it pretty closely since it's been a point of contention for many users.

Reply:
Once we fix the compilation issues, then we can get this information from the CI log. I would actually prefer to have only the following two instantiations:
```c++
instantiate_raft_neighbors_ball_cover(int64_t, float, int64_t, int64_t);
instantiate_raft_neighbors_ball_cover(int64_t, double, int64_t, int64_t);
```